docker/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN pip install --upgrade pip && pip install --upgrade virtualenv && \
     virtualenv /opt/gcp_variant_transforms/venv && \
     . /opt/gcp_variant_transforms/venv/bin/activate && \
     cd /opt/gcp_variant_transforms/src && \
-    pip install --upgrade .
+    pip install --upgrade --no-binary pyvcf,apache-beam[gcp] .
```

Comment:
why apache-beam[gcp] is required? This was deleted recently.

Reply:
This is for making sure these two lib are not installed from binary (so if it's possible use cython).
I saw Asha had a change today related to the new version of apache-beam, but I am not really sure if this change still is needed or not.
@arostamianfar what do your think?

Reply:
I don't see how `--no-binary` is relevant to cython. If you need to specify this, I think there is a bug in how these packages are distributed.

Their configs look alright after all:
https://github.com/apache/beam/blob/master/sdks/python/MANIFEST.in
https://github.com/jamescasbon/PyVCF/blob/master/MANIFEST.in

Reply:
--no-binary causes pip not to use binary packages and instead compile them from their source. 

In the case of pyvcf we definitely know --no-binary makes a difference; it causes the test case fails which indicates pyvcf is using cython.

Reply:
You no longer need to specify apache-beam[gcp] as pre-compiled wheel packages are available as of 2.4.0 (e.g. if you look at what is installed in docker, it is now "apache_beam-2.4.0-cp27-cp27mu-manylinux1_x86_64.whl"). You should keep the pyvcf one though :)

Reply:
Pyc files are included in both cases (building either from source or bin). You can see them in `lib/python2.7/site-packages/vcf/`. And setuptools should install cython version in anyway as you have installed cython already. Therefore, It's not clear why installing pyvcf from bin doesn't work.

Reply:
If you could figure out why it's not working, great! if not, I think we can keep this as is. LGTM!